src/content-feature.js

```diff
@@ -0,0 +1,59 @@
+import { getFeatureSetting, getFeatureSettingEnabled, matchHostname, getFeatureAttr } from './utils'
+
+export default class ContentFeature {
+    constructor (featureName) {
+        this.name = featureName
+        this._args = null
+    }
+
+    /**
+     * @param {import('./utils').Platform} platform
+     */
+    set platform (platform) {
+        this._platform = platform
+    }
+
+    get platform () {
+        return this._platform
+    }
+
+    getFeatureAttr (attrName, defaultValue) {
```

Comment:
I think you already mentioned coming back and adding types to these in a follow up?

Assuming they were untyped before too? if so, fine by me to reduce the size of the initial change 💪🏻